BankSDK/GiniBankSDKExample/Tests/ExtractionFeedbackIntegrationTest.swift

```diff
@@ -49,7 +46,7 @@ class ExtractionFeedbackIntegrationTest: XCTestCase {
         // (subsequent steps are in CaptureResultsDelegateForExtractionsFeedbackTest)
         self.getExtractionsFromGiniBankSDK(delegate: CaptureResultsDelegateForExtractionsFeedbackTest(integrationTest: self, expect: expect))
 
-        wait(for: [expect], timeout: 30)
+        wait(for: [expect], timeout: 60)
```

Comment:
Let's keep 30 sec - it should be enough or maybe we have a special reason for 60 sec?

Reply:
With 30 seconds, locally the success rate was not 100%, with this increase it succeeded all the time. But we still have the problem with the CI, so maybe this is not a valid fix and I have to check the problem elsewhere